libraries/core-react/stories/components/Lists.stories.tsx

```diff
@@ -2,76 +2,85 @@ import React from 'react'
 import { List, ListProps } from '@components'
 import { Meta, Story } from '@storybook/react'
 
-const { ListItem } = List
+const { Item } = List
```

Comment:
No longer needed i presume?

Reply:
It was needed for the default export as `Meta` but changed the syntax slightly